addons/mail/models/mail_thread.py

```diff
@@ -2873,6 +2873,14 @@ def _message_compute_subject(self):
     def _message_create(self, values_list):
         """ Low-level helper to create mail.message records. It is mainly used
         to hide the cleanup of given values, for mail gateway or helpers."""
+        values_list = [
```

Comment:
Is it even possible to have `from`, `to`, `cc`?
It seems like they should raise in `_raise_for_invalid_parameters` as they are not in `_get_message_create_valid_field_names`?
If we need to support them somehow, then I could simply move them to the ignore list.

That's also why I cleaned up the ignore list at the top, it was to be before the raise for invalid.
Otherwise `ValueError: Those values are not supported when posting or notifying: rating_id`.

Reply:
All these keys are failing too:
`ValueError: Those values are not supported when posting or notifying: to`

Is it dead code that I can remove, or I should add them to ignore list?

Reply:
It was related to mailgateway code (sending a bit too much values when injecting emails for message_post), not sure if still relevant or not, don't have much time to check that currently to be honest ^^ .

Reply:
Ah the pop is done after the check ? Ah indeed then it is dead code :D